Sources/Public/Animation/LottieAnimation.swift

```diff
@@ -157,4 +157,7 @@ public final class LottieAnimation: Codable, DictionaryInitializable {
   /// Markers
   let markers: [Marker]?
   let markerMap: [String: Marker]?
+    
+  /// DotLottie configuration to setup the player
+  var dotLottieConfiguration: DotLottieConfiguration?
```

Comment:
This seems reasonable to me. I like the idea of the `DotLottieFile` object returning an array of the `LottieAnimation`s that it contains. That lets us avoid having to add a bunch of new public methods on `LottieAnimationView`, which is nice.